packages/beacon-node/src/chain/lightClient/index.ts

```diff
@@ -517,9 +529,11 @@ export class LightClientServer {
    */
   private async maybeStoreNewBestPartialUpdate(
     syncPeriod: SyncPeriod,
-    syncAggregate: altair.SyncAggregate,
+    syncAggregateBlock: altair.BeaconBlock,
```

Comment:
Can you just add the signature slot as an additional param here?
Passing in the whole block will prob be annoying to deal with future block types.